### PR TITLE
feat: Build MVP of TitleBlock

### PR DESCRIPTION
**Pull request by sebpearce**
## Background

PX is building the Zen version of the Title Block to be rolled out across Perform.

The goal of this PR is to get it to a level of quality where it is ready for use in production, even if not yet fully-featured.

I have fixed a number of bugs that existed on the half-finished previous version of TitleBlockZen.

### CODE REVIEW

File: draft-packages/stories/TitleBlockZen.stories.tsx [hunk 1]
```diff
@@ -296,7 +327,7 @@ export const Performance = () => (
       },
     }}
     avatar={<img alt="avatar image" src={assetUrl("site/empty-state.png")} />}
-    subtitle="Marketing Associate"
+    // subtitle="Director of People Operations and Stuff That's Important And Things"
```
Comment (sampnorris) [inline draft-packages/stories/TitleBlockZen.stories.tsx:330]: Comment

File: draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/MobileActions.scss [hunk 2]
```diff
@@ -19,29 +21,48 @@ $mobile-actions-split-select-square-width: 64px;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   overflow: hidden;
+  // $kz-layout-mobile-actions-drawer-height is 60px but we can't use the var in calc()
+  transform: translateY(calc(100% - 60px));
```
Comment (sampnorris) [inline draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/MobileActions.scss:25]: Yes you can - `calc(100% - #{$kz-layout-mobile-actions-drawer-height})`

File: draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/SecondaryActions.tsx [hunk 3]
```diff
@@ -51,33 +39,36 @@ const SecondaryActions = ({
...
+            </MenuContent>
+          </Menu>
+        )
+      } else {
+        return <Button secondary reversed={reversed} {...a} />
+      }
+    })
+  }
 
   let toolbarItems = stuff
```
Comment (sampnorris) [inline draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/SecondaryActions.tsx:73]: Why not toolbarItems from the start?

File: draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/MobileActions.scss (hunk 2 above)
Comment (sebpearce) [reply draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/MobileActions.scss:25]: OMG

Comment (sebpearce) [reply draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/MobileActions.scss:25]: Damn it, I used to know this

File: draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/SecondaryActions.tsx (hunk 3 above)
Comment (sebpearce) [reply draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/SecondaryActions.tsx:73]: Ah that `stuff` was not meant to be there, cheers

**Pull request merged**